src/views/VirtualDom.ts

```diff
@@ -114,7 +114,6 @@ export default class VirtualDom {
 
     if (contentHeight > frameHeight) {
       // init virtual dom behavior
-      console.log("am i here");
```

Comment:
Where did you go?!

Reply:
Can confirm - 0 tiggers.

Reply:
hrm. This is clearly a problem. Must add tiggers.